QSG-pack/src/main/resources/QSG/bin/qsg.sh

```diff
@@ -0,0 +1,1053 @@
+#!/bin/sh
```

Comment:
What if a user ran the same thing twice, will he get errors like "dupliacate SP" ?, or Is there a place to revert the changes?

Reply:
The setup will be cleaned after every try. Therefore all the users, roles and SPs will be deleted at the end of each run.

Reply:
What will happen if corrupted in the middle?

Reply:
The script will exit and return -1.

Reply:
Does the changes are revert back?

Reply:
The setup will be cleaned up to that point.